repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/NopolRepair.java

```diff
@@ -28,6 +28,7 @@
 import java.util.concurrent.*;
 
 import static fr.inria.spirals.repairnator.process.git.GitHelper.*;
+import static fr.inria.spirals.repairnator.process.files.FileHelper.*;
```

Comment:
If `FileHelper` is not used in this class, the unused import will also break the build because of the checkstyle.

Reply:
It was GitHelper that was unused. I believe only the deleteFile()-method is called from FileHelper.